tests/legacy-cli/e2e/utils/project.ts

```diff
@@ -220,8 +221,12 @@ export async function useCIChrome(projectDir: string = ''): Promise<void> {
   }
 }
 
-export const NgCLIVersion = new SemVer(packages['@angular/cli'].version);
```

Comment:
Since `packages` is now computed from the package tars passed on the CLI there's no guarantee that it has been put into the "global variable"s when this file is first loading, so I've moved it into a method.

Previously in this PR I had this doing `require(../../../....../package.json).version` instead of reading it from the loaded packages. However a) that is not the same as before this PR and b) the git repo version might not be the same as the packages being tested.